### PR TITLE
docs(adt): add link to samples

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/WritableProperty.cs [hunk 1]
```diff
@@ -10,7 +10,12 @@ namespace Azure.DigitalTwins.Core.Serialization
     /// The ModelProperties dictionary on <see cref="DigitalTwinMetadata"/>.
     /// </summary>
     /// <remarks>
+    /// <para>
     /// A writable property is one that the service may request a change for from the device.
+    /// </para>
```
Comment (barustum) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/WritableProperty.cs:15]: para means paragraph?

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/TelemetryOptions.cs [hunk 2]
```diff
@@ -8,6 +8,9 @@ namespace Azure.DigitalTwins.Core.Models
     /// <summary>
     /// The additional information to be used when processing a telemetry request.
     /// </summary>
+    /// <remarks>
+    /// For more samples and information, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/TelemetryOptions.cs:12]: I don't feel 100% on this wording, so if you'd like to propose something better, I'm all ears! 🐰

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/WritableProperty.cs (hunk 1 above)
Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/WritableProperty.cs:15]: https://review.docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments?branch=master

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/TelemetryOptions.cs (hunk 2 above)
Comment (vinagesh) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/TelemetryOptions.cs:12]: I feel it is sufficient to say - For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples"/>

**Review by vinagesh — APPROVED** [review]

**Review by barustum — APPROVED** [review]

**Pull request merged**